src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/EditingPreferencesPane.java

```diff
@@ -203,10 +204,11 @@ public void onClick(ClickEvent event)
       displayPanel.add(checkboxPref("Show indent guides", prefs_.showIndentGuides()));
       displayPanel.add(checkboxPref("Blinking cursor", prefs_.blinkingCursor()));
       displayPanel.add(checkboxPref("Show syntax highlighting in console input", prefs_.syntaxColorConsole()));
+      displayPanel.add(checkboxPref("Different color for console stderr output (requires restart)",
```

Comment:
I think `stderr` is a little too technical for most users to understand -- maybe just "error"?